app/helpers/classifyIntlMessage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function (className) {
```

Comment:
also also, now i'm thinking this should live in `cobuy/lib/`, where re-usable code that has nothing to do with our app goes, which should really become a new module or a pull request to an existing module.